requirements.txt

```
@@ -4,7 +4,7 @@
 #
 #    pip-compile requirements.in
 #
---find-links=https://download.pytorch.org/whl/torch_stable.html
+--find-links https://download.pytorch.org/whl/torch_stable.html
```

```suggestion
--find-links=https://download.pytorch.org/whl/torch_stable.html
```
I need to fix this error so for time being let's add `=`
```
      File "/home/runner/work/obsei/obsei/setup.py", line 38, in get_dependency_links
        for line in lines
      File "/home/runner/work/obsei/obsei/setup.py", line 39, in <listcomp>
        if line.strip().startswith("--find-links")
    IndexError: list index out of range
    ----------------------------------------
WARNING: Discarding file:///home/runner/work/obsei/obsei. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
Error: Process completed with exit code 1.
```

---

done!